cmd/swarm/flags.go

```diff
@@ -153,6 +153,10 @@ var (
 		Name:  "stdin",
 		Usage: "reads data to be uploaded from stdin",
 	}
+	SwarmUploadPublisher = cli.StringFlag{
+		Name:  "publisher",
+		Usage: "Manually specify Publisher",
+	}
```

Comment:
would this be:
- a `publisher` specified in an upload which is then put into the manifest? or
- the fallback publisher we discussed with @vojtechsimetka?

Reply:
this would be 

> a publisher specified in an upload which is then put into the manifest? or

Reply:
thank you for clearing this up